spec/paperclip/io_adapters/file_path_adapter_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe Paperclip::FilePathAdapter do
+  context "a new instance" do
+    context "with normal file" do
+      before do
+        @file_path = fixture_file("5k.png")
+        @file = File.new(@file_path)
+        @file.binmode
+      end
+
+      after do
+        @file.close
+        @subject.close if @subject
+      end
+
+      context 'comparing to original file' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.